roles/keycloak_quarkus/templates/keycloak.conf.j2

```diff
@@ -69,13 +69,12 @@ cache-config-file=cache-ispn.xml
 {% endif %}
 {% endif %}
 
-{% if keycloak_quarkus_proxy_mode is defined and keycloak_quarkus_proxy_mode != "none" %}
-# Deprecated Proxy configuration
-proxy={{ keycloak_quarkus_proxy_mode }}
-{% endif %}
 {% if keycloak_quarkus_proxy_headers is defined and keycloak_quarkus_proxy_headers != "none" %}
```

Comment:
Problem here coming from somewhere else; it seems keycloak_quarkus_proxy_headers is either "", "forwarded" or "xforwarded", but never "none". Can you change here to keycloak_quarkus_proxy_mode != ""

Reply:
good catch, thx; added proper validation